### PR TITLE
Update package itself and dojot-module-logger library version

### PR DESCRIPTION
**Pull request by nunesjoab**

### CODE REVIEW

**Comment by raulnegreiros** [comment]
The change looks great, but you are trying to merge it into the `master`, it should be `development`

File: package-lock.json [hunk 1]
```diff
@@ -1,151 +1,203 @@
...
+  "version": "2.1.0-beta.4",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
     "@dojot/dojot-module": {
-      "version": "0.0.1-beta.1",
-      "resolved": "https://registry.npmjs.org/@dojot/dojot-module/-/dojot-module-0.0.1-beta.1.tgz",
-      "integrity": "sha512-+wntoWl0dxJVbBfcOOtP3XAGc6oCOKugrUQoFd9RMMPHRzq05xBwgGz1kbRrJ461ZsuKVXY+WfhG1VHu94TR9Q==",
+      "version": "0.0.1-beta.5",
+      "resolved": "http://localhost:4873/@dojot%2fdojot-module/-/dojot-module-0.0.1-beta.5.tgz",
```
Comment (raulnegreiros) [inline package-lock.json]: Hum... I think you need to generate it using the `npmjs` not your private registry

**Comment by nunesjoab** [comment]
> The change looks great, but you are trying to merge it into the `master`, it should be `development`

My mistake. I will change the target branch

File: package-lock.json (hunk 1 above)
Comment (nunesjoab) [reply package-lock.json]: Right! Thank you Raul! I will fix this and open a new PR

**Pull request merged**